src/components/forms/AcademicGrantsForm.tsx

```diff
@@ -0,0 +1,594 @@
+            render={({ field: { onChange } }) => (
+              <Field id='applyingAs' label='In which capacity are you applying?' isRequired>
+                <Select
+                  id='applyingAs'
+                  options={APPLYING_AS_OPTIONS}
+                  onChange={option =>
+                    onChange((option as typeof APPLYING_AS_OPTIONS[number]).value)
+                  }
+                  components={{ DropdownIndicator }}
+                  placeholder='Select'
+                  closeMenuOnSelect={true}
+                  selectedOptionColor='brand.option'
+                  chakraStyles={chakraStyles}
+                />
+              </Field>
+            )}
+          />
+
+          <Box display={applyingAs === OTHER ? 'block' : 'none'}>
+            <Fade in={applyingAs === OTHER} delay={0.25}>
+              <TextField id='applyingAsOther' label='If other, please specify' />
+            </Fade>
+          </Box>
+
+          <TextField
+            id='company'
+            label='Institution or Organization'
+            helpText='Name of your university program, team, or organization. If you do not have an organization name, write "N/A"'
+            isRequired
+          />
+
+          <Controller
+            name='country'
+            control={control}
+            render={({ field: { onChange }, fieldState: { error } }) => (
+              <Field
+                id='country'
+                label='Country'
+                helpText='Please indicate the country where the Institution/Lead Investigator is located'
+                error={error}
+                isRequired
+              >
+                <Select
+                  id='country'
+                  options={COUNTRY_OPTIONS}
+                  onChange={option => {
+                    const value = (option as typeof COUNTRY_OPTIONS[number]).value;
+                    onChange(value);
+                  }}
+                  components={{ DropdownIndicator }}
+                  placeholder='Select'
+                  closeMenuOnSelect={true}
+                  selectedOptionColor='brand.option'
+                  chakraStyles={chakraStyles}
+                />
+              </Field>
+            )}
+          />
+
+          <TextField
+            id='countriesTeam'
+            label='If you are a team of distributed researchers, please indicate where your fellow
+            researchers are located'
+            helpText='You can write as many countries as needed'
+            isRequired
+          />
+
+          <Controller
+            name='timezone'
+            control={control}
+            render={({ field: { onChange }, fieldState: { error } }) => (
+              <Field
+                id='timezone'
+                label='Time zone'
+                helpText='Please choose your current time zone to help us schedule calls'
+                error={error}
+                isRequired
+              >
+                <Select
```

Comment:
Select missing the error PageText when no option is selected

Reply:
Same here and the next ones, can you double-check now?